packages/astro/src/vite-plugin-jsx/index.ts

```diff
@@ -137,6 +137,8 @@ export default function jsx({ config, logging }: AstroPluginJSXOptions): Plugin
         const { code: jsxCode } = await esbuild.transform(code, {
           loader: getEsbuildLoader(path.extname(id)) as esbuild.Loader,
           jsx: 'preserve',
+          sourcefile: id,
+          sourcemap: 'inline',
```

Comment:
have you confirmed that this fixed the issue / are JSX stack traces now more accurate with this fix?

Reply:
Yes! I was seeing the correct line/column on JSX errors now